README.md

```diff
@@ -363,6 +363,14 @@ def handler(event, context):
     context.iopipe.mark.measure('expensive operation')
 ```
 
+The trace plugin can also trace your HTTP/HTTPS requests automatically. To enable this feature, set `auto_http` to `True` or set the `IOPIPE_TRACE_AUTO_HTTP` environment variable. For example:
+
+```python
+iopipe = IOpipe(plugins=[TracePlugin(auto_http=True)])
+```
+
+With `auto_http` enabled, you will see traces for any HTTP/HTTPS requests you make within your function on your IOpipe dashboard. Currently this feature only supports the `requests` library, including `boto3` and `botocore` support.
```

Comment:
It seems unusual this doesn't support the standard built-in urllib3... which would also defacto support requests as well?

Reply:
@ewindisch This is meant to be an MVP, requests is the most widely used HTTP in Python by a wide margin. And it monkey patches botocore's vendored version of requests as well. This doesn't support urllib2 or httplib, either. Support could be added, but I don't know any Python devs who would prefer urllib3 over requests (Which is built on top of urllib3).